<ui version="4.0" >
 <class>RestoreForm</class>
 <widget class="QWidget" name="RestoreForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Restore</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelRestore" >
       <property name="font" >
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>681</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnRefresh" >
       <property name="text" >
        <string>Refr&amp;esh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>561</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="labelSelectBackup_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Select a prefix:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="comboBoxPrefixes" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="labelPrefixes" >
       <property name="text" >
        <string>info</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="labelSelectBackup" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Select a backup:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="comboBoxBackupNames" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy" >
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="labelBackupNames" >
       <property name="text" >
        <string>info</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelRestoreType" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Restore type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonFull" >
       <property name="text" >
        <string>F&amp;ull</string>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonCustom" >
       <property name="text" >
        <string>&amp;Custom</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>431</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <widget class="QGroupBox" name="groupBoxCustom" >
     <property name="title" >
      <string>Customize</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="labelSelectItems" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Select files
and directories:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>211</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="treeViewFilesAndFolders" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelRestoreTo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Restore to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonOrigin" >
       <property name="text" >
        <string>Ori&amp;gin</string>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonSpecific" >
       <property name="text" >
        <string>S&amp;pecific</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPath" />
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse" >
       <property name="text" >
        <string>Bro&amp;wse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>291</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestore" >
       <property name="text" >
        <string>Rest&amp;ore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxCustom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>226</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel" >
     <x>79</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSpecific</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>215</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel" >
     <x>634</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSpecific</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnBrowse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>215</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel" >
     <x>902</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
